net-mgmt/networkmanager-proto/Makefile

```diff
@@ -0,0 +1,51 @@
+PORTNAME=	networkmanager-proto
+DISTVERSION=	1.52.0
+CATEGORIES=	net-mgmt
+
+MAINTAINER=	kde@FreeBSD.org
+COMMENT=	Headers for a standard network configuration tool suite
+WWW=		https://networkmanager.dev
+
+LICENSE=	LGPL2
+LICENSE_FILE=	${WRKSRC}/COPYING
+
+LIB_DEPENDS=	libdbus-1.so:devel/dbus \
+		libndp.so:net/libndp \
+		libudev.so:devel/libudev-devd \
+		libuuid.so:misc/e2fsprogs-libuuid
+BUILD_DEPENDS=	xsltproc:textproc/libxslt
+
+USES=		gnome meson perl5 pkgconfig
+USE_GITLAB=     yes
+GL_SITE=        https://gitlab.freedesktop.org
+GL_ACCOUNT=	NetworkManager
+GL_PROJECT=	NetworkManager
+MESON_ARGS=	-Dconcheck=false \
+		-Dcrypto=null \
+		-Ddbus_conf_dir=disabled \
+		-Dintrospection=false \
+		-Dlibaudit=no \
+		-Dlibpsl=false \
+		-Dmodem_manager=false \
+		-Dnm_cloud_setup=false \
+		-Dnmcli=false \
+		-Dnmtui=false \
+		-Dovs=false \
+		-Dpolkit=false \
+		-Dppp=false \
+		-Dreadline=none \
+		-Dselinux=false \
+		-Dsession_tracking=no \
+		-Dsystemd_journal=false \
+		-Dsystemdsystemunitdir=disabled \
+		-Dtests=no
+USE_GNOME=	glib20
+
+NO_BUILD=yes
+
+do-install:
+	${MKDIR} ${STAGEDIR}${PREFIX}/include
+	${CP} -r ${WRKDIR}/NetworkManager-${DISTVERSION}/src/libnm-client-public ${STAGEDIR}${PREFIX}/include/libnm
```

Comment:
This should be replaced with `cd ${WRKSRC}/src/libnm-client-public && ${COPYTREE_SHARE} ...`

Reply:
What comes after `${COPYTREE_SHARE}`?